core/src/test/java/io/questdb/test/cutlass/http/IODispatcherTest.java

```diff
@@ -153,7 +158,7 @@ public void tearDown() throws Exception {
     public void testBadImplicitStrToLongCast() throws Exception {
         getSimpleTester().run(engine -> {
             testHttpClient.assertGet("{\"ddl\":\"OK\"}", "create table tab (value int, when long, ts timestamp) timestamp(ts) partition by day bypass wal;");
-            testHttpClient.assertGet("{\"ddl\":\"OK\"}", "insert into tab values(1, now(), now());"); // it should not be DDL:OK. change me when https://github.com/questdb/questdb/issues/3858 is fixed
+            testHttpClient.assertGet("{\"dml\":\"OK\"}", "insert into tab values(1, now(), now());"); // it should not be DDL:OK. change me when https://github.com/questdb/questdb/issues/3858 is fixed
```

Comment:
The TODO can be removed now:
```suggestion
            testHttpClient.assertGet("{\"dml\":\"OK\"}", "insert into tab values(1, now(), now());");
```